libs/application/templates/driving-license/src/fields/QualityPhoto/hooks/getQualityPhoto.ts

```diff
@@ -0,0 +1,60 @@
+import { Application } from '@island.is/application/core'
+import { isRunningOnEnvironment } from '@island.is/shared/utils'
+import { QUALITY_PHOTO } from './queries.graphql'
+import { useQuery } from '@apollo/client'
+
+export interface QualityPhotoType {
+  qualityPhoto: string | null
+  success: boolean
+}
+
+const YES = 'yes'
+
+type YesOrNo = 'yes' | 'no'
+
+interface DataProviderFakeData {
+  useFakeData?: YesOrNo
+  qualityPhoto?: YesOrNo
+}
+
+type HasQualityPhotoData = {
+  data: {
+    hasQualityPhoto: boolean
+  }
+}
+
+export const getQualityPhoto = (application: Application): QualityPhotoType => {
+  // If running locally or on dev allow for fake data
+  if (isRunningOnEnvironment('local') || isRunningOnEnvironment('dev')) {
+    const fakeData = application.answers.fakeData as
```

Comment:
using as is an anti pattern.. to enforce type safety for answers, esp. when it comes to undefined etc, we should try to always use
```typescript
const value = getValueViaPath<'yes'|'no'>(application.answers, 'fakeData')
```
the resulting type is 'yes'|'no'|undefined, which ensures that the rest of the code is correct